src/app/pages/news/9.3/01-scaling.md

```diff
@@ -0,0 +1,24 @@
+## Vertical Scaling {#scaling}
+
+It was never easier to scale Axon Ivy Engine. 
+
+- **Container**: Use the container technology of your choice to scale Axon Ivy Engine vertically to match the current load of your clients.
```

Comment:
I'd involve more technology buss-words: e.g docker, kubernetes cluster, ...

Reply:
'vertical scaling' doesn't even speak to me ... get a broader audience by selling heavy load scenarios